test/evaluation/subjectIsIssuerEvaluationHandler.spec.ts

```diff
@@ -59,7 +69,7 @@ describe('evaluate', () => {
 
   it('should return error if subject_is_issuer is not verified', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-subject-is-issuer.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-subject-is-issuer.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-subject-is-issuer.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.